--- prelink/src/Makefile.am.orig	2003-11-18 11:32:00.000000000 +0100
+++ prelink/src/Makefile.am	2004-06-04 13:22:11.331762656 +0200
@@ -21,7 +21,7 @@
 		  verify.c md5.c md5.h sha.c sha.h			     \
 		  $(common_SOURCES) $(arch_SOURCES)
 prelink_LDADD = @LIBGELF@
-prelink_LDFLAGS = -all-static
+prelink_LDFLAGS =
 
 execstack_SOURCES = execstack.c $(common_SOURCES) $(arch_SOURCES)
 
--- prelink/src/Makefile.am.orig	2003-11-18 11:32:00.000000000 +0100
+++ prelink/src/Makefile.am	2004-06-04 13:22:11.331762656 +0200
@@ -21,7 +21,7 @@
 		  verify.c md5.c md5.h sha.c sha.h			     \
 		  $(common_SOURCES) $(arch_SOURCES)
 prelink_LDADD = @LIBGELF@
-prelink_LDFLAGS = -all-static
+prelink_LDFLAGS =
 
 execstack_SOURCES = execstack.c $(common_SOURCES) $(arch_SOURCES)
 
